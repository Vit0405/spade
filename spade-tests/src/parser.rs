use crate::snapshot_error;

snapshot_error! {
    stage_outside_pipeline,
    "
    entity main(x: X) -> int<8> {
        reg;
    }
    "
}

snapshot_error! {
    register_count_is_required,
    "
    pipeline(3) main(x: X) -> int<8> {
        reg *;
    }
    "
}
