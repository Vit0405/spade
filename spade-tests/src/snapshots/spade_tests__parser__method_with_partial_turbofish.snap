---
source: spade-tests/src/parser.rs
---
fn test() {
    a.field::
}


error: Expected $ or <
  ┌─ testinput:3:1
  │
2 │     a.field::
  │            -- :: after an method is used to specify type parameters
3 │ }
  │ ^ Expected $ or <
